### ISSUE TITLE
Grid with title empty

### ISSUE THREAD

**Issue by meriturva** [telerik/kendo-angular#334]
I have created a simple plunker: http://plnkr.co/edit/sxJBufp0hVlmqlHGfJ2e

Main problem is that i cannot set header title empty.

If i put attribute title= null or empty string field name is showed.
If i use custom kendoGridHeaderTemplate i have link concerning sort feature on header dom.

Any advice how to have header empty?
Thanks.

**Comment by PeterStaev** [comment]
This has been around since kendo for jQuery. The workaround is to use a space, so `title=" "` 😄

**Comment by meriturva** [comment]
Thanks @PeterStaev  but i think to use space to disable title is not so correct. am i right?

**Comment by tsvetomir** [comment]
We should be able to make a distinction between undefined and empty string. Will look into it.

**Comment by tsvetomir** [comment]
This bug should be fixed in version 0.26.4 of the Grid. Please note that:

* The following settings will produce an empty title:
```html
<kendo-grid-column field="ProductName" title="">
</kendo-grid-column>
<kendo-grid-column field="ProductName" [title]="null">
</kendo-grid-column>
```

* Setting `undefined` will still display the field name:
```html
<kendo-grid-column field="ProductName" [title]="">
</kendo-grid-column>
```

* The group headers will still use the field name if an empty title is set.
